config/core/300-secret.yaml

```diff
@@ -13,17 +13,17 @@
 # limitations under the License.
 
 apiVersion: v1
-kind: ConfigMap
+kind: Secret
 metadata:
-  name: config-network
+  name: serving-certs-ctrl-ca
```

Comment:
What does `ctrl` stand for here?

Reply:
I also feel strange but It is a control-protocol's regulation - https://github.com/knative-sandbox/control-protocol/blob/main/pkg/certificates/reconciler/controller.go#L37-L40

Reply:
Ah, great. Maybe a comment with a link to the `control-protocol` repo?

Reply:
Sure, added the comment.